server/services/licenceService.ts

```diff
@@ -240,11 +240,11 @@ export default class LicenceService {
     return this.licenceApiClient.updateBespokeConditions(id, requestBody, user)
   }
 
-  async updateStatus(id: string, newStatus: LicenceStatus, user: User): Promise<void> {
+  async updateStatus(id: string, newStatus: LicenceStatus, user?: User): Promise<void> {
     const requestBody = {
       status: newStatus,
-      username: user.username,
-      fullName: user.displayName,
+      username: user?.username || 'SYSTEM',
+      fullName: user?.displayName || 'SYSTEM',
```

Comment:
These are used for auditing, who updated the status? In this instance, it's the system who updates the status.